test-capi failed when running cargo ci locally

When running `cargo ci` locally on my Linux machine, `test-capi` failed. I don't know why the failure is not show up on CI. 

@Manishearth Any ideas?

The log snippet from `cargo ci`:
```
[cargo-make] INFO - Running Task: test-capi
cargo build
   Compiling writeable v0.2.0 (/home/aethanyc/Projects/icu4x/utils/writeable)
   Compiling icu_locid v0.2.0 (/home/aethanyc/Projects/icu4x/components/locid)
   Compiling fixed_decimal v0.2.0 (/home/aethanyc/Projects/icu4x/utils/fixed_decimal)
   Compiling icu_provider v0.2.0 (/home/aethanyc/Projects/icu4x/provider/core)
   Compiling icu_plurals v0.2.0 (/home/aethanyc/Projects/icu4x/components/plurals)
   Compiling icu_decimal v0.2.0 (/home/aethanyc/Projects/icu4x/components/decimal)
   Compiling icu_provider_fs v0.2.0 (/home/aethanyc/Projects/icu4x/provider/fs)
   Compiling icu_capi v0.1.0 (/home/aethanyc/Projects/icu4x/ffi/capi)
    Finished dev [unoptimized + debuginfo] target(s) in 0.88s
gcc test.c ../../../../target/debug/libicu_capi.a -ldl -lpthread -lm -g
./a.out
Plural Category 5 (should be 3)
make: *** [Makefile:25: test] Error 1
[cargo-make] ERROR - Error while executing command, exit code: 2
[cargo-make] WARN - Build Failed.
```

--- a/ffi/capi/examples/pluralrules/test.c
+++ b/ffi/capi/examples/pluralrules/test.c
@@ -22,8 +22,7 @@ int main() {
     }
     ICU4XPluralRules* rules = plural_result.rules;
 
-    ICU4XPluralOperands op;
-    op.i = 3;
+    ICU4XPluralOperands op = { .i = 3 };
 
     ICU4XPluralCategory cat = icu4x_plural_rules_select(rules, &op);
 

